### PR TITLE
Add feature to return only non-expired entries

### PR DESCRIPTION
**Pull request by schrauger**
This new feature lets the user return only entries that are current and valid. It excludes all entries that are expired (expiry date has happened).

It adds a checkbox to the General tab of KeePassHttp Options. I made sure that the checkboxes and other elements were pushed down by 23 pixels, keeping the padding consistent.

The date check is copied from KeePass's own source code, so it should be consistent with the main app's definition of an expired entry.

I also recompiled the plgx file to include the new feature. I tested it with both Firefox (PassIFox) and Chrome (chromeIPass) on Linux Mint 18. **Edit** I removed the plgx from the branch, as it looks like you don't want that included (per the other pull requests' comments).

I did not add a test.

### CODE REVIEW

File: KeePassHttp/Handlers.cs [hunk 1]
```diff
@@ -234,6 +234,23 @@ private IEnumerable<PwEntryDatabase> FindMatchingEntries(Request r, Aes aes)
                 result = from e in result where filterSchemes(e.entry) select e;
             }
 
+            Func<PwEntry, bool> hideExpired = delegate(PwEntry e)
+            {
+			    DateTime dtNow = DateTime.UtcNow;
```
Comment (pfn) [inline KeePassHttp/Handlers.cs]: fix indentation

**Comment by pfn** [comment]
looks good, just fix the minor space thing and I'll merge.

Thanks for the PR

**Comment by schrauger** [comment]
I changed the one line that had tab indents to space indents.

**Comment by pfn** [comment]
thank you

**Comment by schrauger** [comment]
Fixes #198

**Comment by schrauger** [comment]
I just realized that when I removed the changed plgx file, it removed it completely. I imagine you'll recompile the file yourself, but right now, the file does not exist in the master branch.

**Pull request merged**